documentation/src/main/asciidoc/titles/encoding/encoding.asciidoc

```diff
@@ -0,0 +1,24 @@
+//Directory attributes
+:stories: ../../stories
+:topics: ../topics
+:imagesdir: ../{topics}/images
+
+//Community doc attributes
+include::../{topics}/attributes/community-attributes.adoc[]
+
+//Community title attributes
+:toc2:
+:icons: font
+:toclevels: 3
+:numbered:
+:sectanchors:
+:stem: asciimath
+
+//Title
+= Marshalling and Encoding Data
+
+{brandname} caches are flexible and can store key/value pairs in different formats.
```

Comment:
I think we should remove "are flexible and"

Reply:
good call. fixed.